src/consensus/consensus.h

```diff
@@ -1,16 +0,0 @@
-// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
...unless we merge something like #6009 first or we keep the new functions (something like https://github.com/jtimon/bitcoin/commit/c59f170fd15688499f7f6291e2ef0cf18a43fcc0#diff-cbe22f30d7e480617350ef6ceca97d0cR12 ) here.
It took very long to create this consensus/consensus.h little step forward, it seems very wasteful to remove all those includes (when most of them will have to be reintroduced later).